docs/user-guide/system-admin/configuration/infinispan.md

```diff
@@ -12,14 +15,38 @@ The Infinispan configuration will be located inside the following module in `sta
 
 Keep in mind that the module version may vary depending on your JBoss version.
 
-### Configuration for Internal data caches
+### Hibernate Cache
+
+The following is the recommended configuration for the Hibernate cache:
```

Comment:
This configuration just seems to duplicate jboss/wildfly standard configuration, and this section should probably be omitted so that we don't have to maintain it.